README.md

```diff
@@ -248,7 +248,7 @@ function Login() {
         </div>
         <button type="submit">Submit{state.loading ? '...' : null}</button>
       </form>
-      {state.error ? <div role="alert">{state.error.message}</div> : null}
+      {state.error ? <div role="alert">{state.error}</div> : null}
```

Comment:
Hi, I have a question about the above syntax.

Is it some convenience to handle show/hidden a component like in README.md?
`{ isTrue ? <Component /> : null }` than `{ isTrue && <Component /> }` (shorter a way)

Reply:
I could write a blog post about this for the number of times I talk about it 😅

Basically I've been bitten by this before:

```
{contacts.length && <RenderContactList contacts={contacts} />}
```

If `contacts.length` is `0` then you'll render `0`. No good. Shipped that to prod at PayPal. So I never do that and instead I always go with the more explicit version:

```
{contacts.length ? <RenderContactList contacts={contacts} /> : null}
```

But you do whatever you like :)